crashes: increment the number of tracked ones

matthiaskrgr opened the pull request:
r? @jieyouxu

matthiaskrgr committed:
crashes: increment the number of tracked ones

matthiaskrgr commented:
@bors rollup=iffy

jieyouxu commented:
Thanks! @bors r+ rollup

bors commented:
:pushpin: Commit ce677681562ca15518c697097b4a77d0f68e2983 has been approved by `jieyouxu`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=jieyouxu ce677681562ca15518c697097b4a77d0f68e2983 -->
<!-- homu: {"type":"Approved","sha":"ce677681562ca15518c697097b4a77d0f68e2983","approver":"jieyouxu","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->

bors commented:
:evergreen_tree: The tree is currently [closed](https://github.com/rust-lang/rust/pull/125576#issuecomment-2132313643) for pull requests below priority 100. This pull request will be tested once the tree is reopened.

workingjubilee commented:
@bors r-

Conflicts with https://github.com/rust-lang/rust/pull/125466

workingjubilee commented on tests/crashes/125432.rs:
Please delete this.

workingjubilee commented on tests/crashes/125520.rs:
This one too, please.

workingjubilee commented on tests/crashes/125370.rs:
If you would.

workingjubilee reviewed (commented):
This has even more conflicts, actually, a bunch of these crashes need to go away:

https://github.com/rust-lang/rust/pull/125592#issuecomment-2132417811

rust-log-analyzer commented:
The job **`x86_64-gnu-llvm-17`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/9248357402/job/25438543014) [(plain)](https://github.com/rust-lang/rust/commit/ce677681562ca15518c697097b4a77d0f68e2983/checks/25438543014/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
#16 exporting to docker image format
#16 sending tarball 29.1s done
#16 DONE 42.5s
##[endgroup]
Setting extra environment values for docker:  --env ENABLE_GCC_CODEGEN=1 --env GCC_EXEC_PREFIX=/usr/lib/gcc/
[CI_JOB_NAME=x86_64-gnu-llvm-17]
---
sccache: Starting the server...
##[group]Configure the build
configure: processing command line
configure: 
configure: build.configure-args := ['--build=x86_64-unknown-linux-gnu', '--llvm-root=/usr/lib/llvm-17', '--enable-llvm-link-shared', '--set', 'rust.thin-lto-import-instr-limit=10', '--set', 'change-id=99999999', '--enable-verbose-configure', '--enable-sccache', '--disable-manage-submodules', '--enable-locked-deps', '--enable-cargo-native-static', '--set', 'rust.codegen-units-std=1', '--set', 'dist.compression-profile=balanced', '--dist-compression-formats=xz', '--disable-dist-src', '--release-channel=nightly', '--enable-debug-assertions', '--enable-overflow-checks', '--enable-llvm-assertions', '--set', 'rust.verify-llvm-ir', '--set', 'rust.codegen-backends=llvm,cranelift,gcc', '--set', 'llvm.static-libstdcpp', '--enable-new-symbol-mangling']
configure: target.x86_64-unknown-linux-gnu.llvm-config := /usr/lib/llvm-17/bin/llvm-config
configure: llvm.link-shared     := True
configure: rust.thin-lto-import-instr-limit := 10
configure: change-id            := 99999999
---

running 227 tests
........................................................................................  88/227
........................................................................................ 176/227
....................2024-05-27T02:52:04.475082Z ERROR compiletest::runtest: fatal error, panic: "test no longer crashes/triggers ICE! Please give it a mearningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit."
[crashes] tests/crashes/125432.rs ... F
[crashes] tests/crashes/125432.rs ... F
2024-05-27T02:52:04.490652Z ERROR compiletest::runtest: fatal error, panic: "test no longer crashes/triggers ICE! Please give it a mearningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit."
[crashes] tests/crashes/125520.rs ... F
[crashes] tests/crashes/125520.rs ... F
......................2024-05-27T02:52:04.970607Z ERROR compiletest::runtest: fatal error, panic: "test no longer crashes/triggers ICE! Please give it a mearningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit."
[crashes] tests/crashes/125370.rs ... F
......

failures:
failures:

---- [crashes] tests/crashes/125432.rs stdout ----

error: test no longer crashes/triggers ICE! Please give it a mearningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit.
thread '[crashes] tests/crashes/125432.rs' panicked at src/tools/compiletest/src/runtest.rs:2830:9:
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- [crashes] tests/crashes/125520.rs stdout ----


error: test no longer crashes/triggers ICE! Please give it a mearningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit.
thread '[crashes] tests/crashes/125520.rs' panicked at src/tools/compiletest/src/runtest.rs:2830:9:

---- [crashes] tests/crashes/125370.rs stdout ----


error: test no longer crashes/triggers ICE! Please give it a mearningful name, add a doc-comment to the start of the test explaining why it exists and move it to tests/ui or wherever you see fit.
thread '[crashes] tests/crashes/125370.rs' panicked at src/tools/compiletest/src/runtest.rs:2830:9:


failures:
    [crashes] tests/crashes/125432.rs
```

</details>

jieyouxu commented:
> lol @jieyouxu fwiw if I set something to rollup=iffy there is usually a good reason for it:D

Oh sorry, I missed the rollup=iffy comment...

matthiaskrgr committed:
remove fixed crashes, add fixed crashes to tests, add new cashed found in the meantime

matthiaskrgr commented:
I've added the last couple of new ices, removed duplicates, added a fixed one as ui test and update one ui test to link to the corresponding ice ticket.
lets give this another go
@bors r=jieyouxu

bors commented:
:pushpin: Commit 894386a2b9a5dc3e8534fa63d4b6364c6907d715 has been approved by `jieyouxu`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=jieyouxu 894386a2b9a5dc3e8534fa63d4b6364c6907d715 -->
<!-- homu: {"type":"Approved","sha":"894386a2b9a5dc3e8534fa63d4b6364c6907d715","approver":"jieyouxu","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->

bors commented:
:hourglass: Testing commit 894386a2b9a5dc3e8534fa63d4b6364c6907d715 with merge 37b74b304b9de97134350f7f081bc084b44c59e6...
<!-- homu: {"type":"BuildStarted","head_sha":"894386a2b9a5dc3e8534fa63d4b6364c6907d715","merge_sha":"37b74b304b9de97134350f7f081bc084b44c59e6"} -->

matthiaskrgr commented:
@bors r=jieyouxu

bors commented:
:pushpin: Commit bae945201f72cf548b3b2d8cfd545812cccfae4a has been approved by `jieyouxu`

It is now in the [queue](https://bors.rust-lang.org/homu/queue/rust) for this repository.

<!-- @bors r=jieyouxu bae945201f72cf548b3b2d8cfd545812cccfae4a -->
<!-- homu: {"type":"Approved","sha":"bae945201f72cf548b3b2d8cfd545812cccfae4a","approver":"jieyouxu","queue":"https://bors.rust-lang.org/homu/queue/rust"} -->

bors commented:
:hourglass: Testing commit bae945201f72cf548b3b2d8cfd545812cccfae4a with merge 9081198f6db37165ab63b7001b8ffd61f7f69e20...
<!-- homu: {"type":"BuildStarted","head_sha":"bae945201f72cf548b3b2d8cfd545812cccfae4a","merge_sha":"9081198f6db37165ab63b7001b8ffd61f7f69e20"} -->

rust-log-analyzer commented:
The job **`aarch64-gnu`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/9261073818/job/25475884940) [(plain)](https://github.com/rust-lang-ci/rust/commit/9081198f6db37165ab63b7001b8ffd61f7f69e20/checks/25475884940/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
test [ui] tests/ui/const-generics/generic_const_exprs/ice-predicates-of-no-entry-found-for-key-119275.rs ... ok
test [ui] tests/ui/const-generics/generic_const_exprs/inline-const-in-const-generic-defaults.rs ... ok
test [ui] tests/ui/const-generics/generic_const_exprs/elaborate-trait-pred.rs ... ok
test [ui] tests/ui/const-generics/generic_const_exprs/issue-100360.rs ... ok
test [ui] tests/ui/const-generics/generic_const_exprs/ice-125520-layout-mismatch-mulwithoverflow.rs ... ok
test [ui] tests/ui/const-generics/generic_const_exprs/from-sig.rs ... ok
test [ui] tests/ui/const-generics/generic_const_exprs/issue-105608.rs ... ok
test [ui] tests/ui/const-generics/generic_const_exprs/issue-105257.rs ... ok
test [ui] tests/ui/const-generics/generic_const_exprs/issue-102074.rs ... ok
---
##[endgroup]
[TIMING] core::build_steps::tool::ToolBuild { compiler: Compiler { stage: 0, host: aarch64-unknown-linux-gnu }, target: aarch64-unknown-linux-gnu, tool: "linkchecker", path: "src/tools/linkchecker", mode: ToolBootstrap, source_type: InTree, extra_features: [], allow_features: "" } -- 0.222
[TIMING] core::build_steps::tool::Linkchecker { compiler: Compiler { stage: 0, host: aarch64-unknown-linux-gnu }, target: aarch64-unknown-linux-gnu } -- 0.004
##[group]Testing stage0 Linkcheck (aarch64-unknown-linux-gnu)
##[error]The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.

Session terminated, killing shell...::group::Clock drift check
  network time: Tue, 28 May 2024 00:01:52 GMT
##[endgroup]
checked links in: 31.6s
number of HTML files scanned: 38518
```

</details>

bors commented:
:broken_heart: Test failed - [checks-actions](https://github.com/rust-lang-ci/rust/actions/runs/9261073818/job/25476767022)
<!-- homu: {"type":"BuildFailed","builder_url":"https://github.com/rust-lang-ci/rust/actions/runs/9261073818/job/25476767022","builder_name":"checks-actions"} -->

workingjubilee commented:
blaming this one on the internet

@bors retry

bors commented:
:hourglass: Testing commit bae945201f72cf548b3b2d8cfd545812cccfae4a with merge 71213fd60700478ac9ddef2cb4ef43b7bb660a5b...
<!-- homu: {"type":"BuildStarted","head_sha":"bae945201f72cf548b3b2d8cfd545812cccfae4a","merge_sha":"71213fd60700478ac9ddef2cb4ef43b7bb660a5b"} -->

bors commented:
:sunny: Test successful - [checks-actions](https://github.com/rust-lang-ci/rust/actions/runs/9261657701/job/25479937066)
Approved by: jieyouxu
Pushing 71213fd60700478ac9ddef2cb4ef43b7bb660a5b to master...
<!-- homu: {"type":"BuildCompleted","approved_by":"jieyouxu","base_ref":"master","builders":{"checks-actions":"https://github.com/rust-lang-ci/rust/actions/runs/9261657701/job/25479937066"},"merge_sha":"71213fd60700478ac9ddef2cb4ef43b7bb660a5b"} -->

rust-timer commented:
Finished benchmarking commit (71213fd60700478ac9ddef2cb4ef43b7bb660a5b): [comparison URL](https://perf.rust-lang.org/compare.html?start=84b40fc908c3adc7e0e470b3fbaa264df0e122b8&end=71213fd60700478ac9ddef2cb4ef43b7bb660a5b&stat=instructions:u).

### Overall result: no relevant changes - no action needed


@rustbot label: -perf-regression



### [Instruction count](https://perf.rust-lang.org/compare.html?start=84b40fc908c3adc7e0e470b3fbaa264df0e122b8&end=71213fd60700478ac9ddef2cb4ef43b7bb660a5b&stat=instructions:u)
This benchmark run did not return any relevant results for this metric.

### [Max RSS (memory usage)](https://perf.rust-lang.org/compare.html?start=84b40fc908c3adc7e0e470b3fbaa264df0e122b8&end=71213fd60700478ac9ddef2cb4ef43b7bb660a5b&stat=max-rss)
<details>
<summary>Results (primary 2.6%)</summary>

This is a less reliable metric that may be of interest but was not used to determine the overall result at the top of this comment.

|                                    | mean | range        | count |
|:----------------------------------:|:----:|:------------:|:-----:|
| Regressions ❌ <br /> (primary)    | 2.6% | [2.6%, 2.6%] | 1     |
| Regressions ❌ <br /> (secondary)  | -    | -            | 0     |
| Improvements ✅ <br /> (primary)   | -    | -            | 0     |
| Improvements ✅ <br /> (secondary) | -    | -            | 0     |
| All ❌✅ (primary)                 | 2.6% | [2.6%, 2.6%] | 1     |
</details>

### [Cycles](https://perf.rust-lang.org/compare.html?start=84b40fc908c3adc7e0e470b3fbaa264df0e122b8&end=71213fd60700478ac9ddef2cb4ef43b7bb660a5b&stat=cycles:u)
This benchmark run did not return any relevant results for this metric.

### [Binary size](https://perf.rust-lang.org/compare.html?start=84b40fc908c3adc7e0e470b3fbaa264df0e122b8&end=71213fd60700478ac9ddef2cb4ef43b7bb660a5b&stat=size:linked_artifact)
This benchmark run did not return any relevant results for this metric.

**Bootstrap**: 669.323s -> 668.641s (-0.10%)
**Artifact size**: 318.28 MiB -> 318.24 MiB (-0.01%)
